### ISSUE TITLE
Getter and Setter detection includes non property methods

### ISSUE THREAD

**Issue by danhagberg** [danhagberg/pojot#13]
Test detection is currently including gutter and setter methods due to them starting with the string "get" or "set".  This is true even if there is no property directly tied to the methods.  Pojot only tests property setters and getters and should exclude methods that are not tied to a property in the class under test.

Example:
``` java
public class SetterGetterForNonPropClass {
    private String testString1;

    public String getTestString1() {
        return testString1;
    }

    public void setTestString1(String testString1) {
        this.testString1 = testString1;
    }

    public void setNonProp(String value) {
        testString1 = value;
    }
```
This will result in the error:
net.digitaltsunami.pojot.TestAidException: Exception during setter test: net.digitaltsunami.pojot.testsubject.SetterGetterForNonPropClass.setNonProp
...
Caused by: java.lang.NoSuchFieldException: nonProp
	at java.lang.Class.getDeclaredField(Class.java:2057)
	at net.digitaltsunami.pojot.SetterTestRunner.testSetter(SetterTestRunner.java:77)

**Comment by danhagberg** [comment]
Note that this can be worked around by excluding the field: 
```java
    @Test
    public void testIgnoresNonPropMethods() throws IntrospectionException {
        List<String> errors = new TestAid<SetterGetterForNonPropClass>(SetterGetterForNonPropClass.class)
                .excludeFields("nonProp")
                .includeAllInEquals()
                .validate();
        assertTrue(errors.isEmpty(), errors.toString());
    }
}
```

**Comment by danhagberg** [comment]
Issue resolved with release 1.2.2.  
Added logic to skip setter and getter methods when there is no corresponding property
